### DISCUSSION TITLE
Better Notifications

### DISCUSSION THREAD

**Discussion by ryanfortner** [community/community#8307] [Mobile]
Right now, you can only get app notifications if you're mentioned in a project. Could there be a feature where you can get notifications for watched repositories, and maybe even for the github activity feed?

**Comment by eliperkins** [comment]
Thanks for the feedback @ryanfortner ! We're iterating on push notifications all the time, including launching [push notifications for pull request reviews](https://github.blog/changelog/2021-11-23-push-notifications-for-pull-request-review-activities-on-github-mobile/) in recent weeks.

I'd love to hear of what other specific actions on watched repositories you'd like push notifications for! I can take your feedback to the team to see what we can do.

**Comment by ryanfortner** [comment]
> I'd love to hear of what other specific actions on watched repositories you'd like push notifications for! I can take your feedback to the team to see what we can do.

Hi @eliperkins! I would love to get app notifications for new releases on watched repositories, as well as newly opened issues and pull requests.
